### DISCUSSION TITLE
Deleting Scenes and  Loading Avatar

### DISCUSSION THREAD

**Discussion by Powerstephan** [mozilla/hubs#5568] [Hubs Cloud Questions]
Hello,

I have some issues inside admin panel.

I am not able to delete any existing scenes :( 

And also I have importer some of the hubs avatars. I can see them inside of admin panel, but if I join any room, I can't select any avatar :(

**Comment by pattersonbl2** [comment]
This is a known issue in hubs. We don't have a timeline for when this will be fixed

**Reply by Powerstephan** [reply]
Both issues? Scenes and avatars?
